src/Components/WordTerm.js

```diff
@@ -1,25 +1,45 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
 
-export default function WordTerm({ word, def }) {
+export default function WordTerm({ word, def, links }) {
+  const linksArray = links.split('\n');
+  linksArray.pop();
```

Comment:
If this is supposed to pop the last element if there is an extra line break, I think it'd be good to add an extra check to make sure that the link is empty! It's possible that they will forget to add an extra line break after. It's also possible that they add extras, so maybe it would be good to filter the array to get rid of elements that are just white space.